Mathlib/Probability/Moments/Covariance.lean

```diff
@@ -46,6 +44,20 @@ scoped notation "cov[" X ", " Y "; " μ "]" => ProbabilityTheory.covariance X Y
 according to the volume measure. -/
 scoped notation "cov[" X ", " Y "]" => cov[X, Y; MeasureTheory.MeasureSpace.volume]
 
+lemma covariance_eq [IsProbabilityMeasure μ] (hX : MemLp X 2 μ) (hY : MemLp Y 2 μ) :
```

Comment:
```suggestion
lemma covariance_eq_sub [IsProbabilityMeasure μ] (hX : MemLp X 2 μ) (hY : MemLp Y 2 μ) :
```
Perhaps this is a slightly more informative name? The variance equivalent is `variance_def'`, which is not very informative either.